components/collective-navbar/index.js

```diff
@@ -586,6 +650,9 @@ const CollectiveNavbar = ({
                 collective={collective}
                 callsToAction={callsToAction}
                 hiddenActionForNonMobile={mainAction?.type}
+                mainAction={mainAction?.type}
+                getActions={getActions}
```

Comment:
> Overall putting the buttons in `CollectiveNavbarActionsMenu` looks like too much responsibility, because it's not just a menu anymore it's now, it's a ActionsButtonsWithMenu. But if it's simpler to do it that way, we should at least make its props clearer:

I chose to render the second CTA button in the `ActionsMenu` component because when we're on the mobile menu, even if we have only 2 CTAs that are buttons, we still have to display the Actions menu in some form, which I think it easier within the component: https://github.com/opencollective/opencollective-frontend/blob/964bb1f28c41f4f3b4a9071bb1f1af2ef06cd050/components/collective-navbar/ActionsMenu.js#L213

#### tablet
![Screenshot 2021-01-27 at 13 51 06](https://user-images.githubusercontent.com/37520401/106000531-d0de4f80-60a6-11eb-8c32-a5520803d5a5.png)

#### mobile
![Screenshot 2021-01-27 at 13 51 34](https://user-images.githubusercontent.com/37520401/106000537-d20f7c80-60a6-11eb-8a16-a0593aa467f6.png)

I'm a bit confused about your suggested changes... passing down the actions function to `ActionsMenu` component is something I've done a lot in other components. What makes it confusing here? Also all the logic we use to figure out if we need a second action button is in `ActionsMenu` so we wouldn't be passing `secondaryAction` down as a prop from `index.js`.

I'd suggest keeping things mostly the same but can rename the function we pass down to be more clear, like `getSecondaryAction={getMainAction}` (although this is why I had renamed the function to `getActions` in the first place since it's being used for both main and secondary).

Reply:
Makes sense to put that in `ActionsMenu` then. 

> Passing down the actions function to ActionsMenu component is something I've done a lot in other components. What makes it confusing here?

It adds complexity: why pass a getter for a value when you could pass this value directly? It's also about consistency: since you pass an action for the primary button, why not do the same for the second one?

The props proposed above intend to improve consistency, without increasing the complexity. All we have to do is:

```jsx
const actionsArray = Object.keys(pickBy(callsToAction, Boolean));
const mainAction = getMainAction(collective, isAdmin, actionsArray);
const secondAction = getMainAction(collective, isAdmin, without(actionsArray, mainAction));

return (
  <CollectiveNavbarActionsMenu
    collective={collective}
    mainAction={mainAction?.type}
    secondaryAction={secondaryAction?.type}
    dropdownActions={without(actionsArray, mainAction, secondAction)}
  />
```

> I'd suggest keeping things mostly the same but can rename the function we pass down to be more clear, like getSecondaryAction={getMainAction} (although this is why I had renamed the function to getActions in the first place since it's being used for both main and secondary). 

Since `getMainAction` takes an action list, its prototype reads as "get the main action from this list". So it's fine to keep the same name for the secondary action - the function itself has no knowledge about what's a primary or a secondary action, it just returns the main action form a given list.